lib/hint/rubocop_style/version.rb

```diff
@@ -1,5 +1,5 @@
 module Hint
   module RubocopStyle
-    VERSION = '0.2.0'.freeze
+    VERSION = '0.3.0'.freeze
```

Comment:
Yeah I'm not sure. I was going back and forth on what to do with numbering. I guess these could all be patch level bug fixes.